sw/device/tests/sim_dv/otp_ctrl_lc_signals_test.c

```diff
@@ -93,14 +99,14 @@ static void init_peripherals(void) {
  * access to fail with a DAI access error. If exp_result is set to
  * kExpectPassed, the test expects the test to pass without errors.
  */
-static void otp_write_test(uint32_t address, const uint8_t *buffer,
-                           uint32_t size, exp_test_result_t exp_result) {
+static void otp_write_test(dif_otp_ctrl_partition_t partition, uint32_t address,
```

Comment:
Do we need to tweak the comment above? I don't think this is guaranteed to be the SECRET2 partition any more? Similar for `otp_read_test` below.

Reply:
Thanks for catching this! Fixed